system_tests/multigas_dump_test.go

```diff
@@ -50,29 +40,24 @@ func TestMultigasCollectorFromNode(t *testing.T) {
 		rcpt, err := builder.L2.EnsureTxSucceeded(tx)
 		require.NoError(t, err)
 
-		want[tx.Hash()] = rcpt.GasUsed
+		txs[tx.Hash()] = rcpt.GasUsed
```

Comment:
You already have the receipt in line 40.
```
		rcpt, err := builder.L2.EnsureTxSucceeded(tx)
```
Could you get the multigas from there, and avoid restarting the node?

Reply:
I could, but I restart the node to verify the receipt was actually persisted in the DB and survives a cold start

Reply:
I kind of agree with @gligneul here.
At that point, you're making the test longer and more complicated just to ensure that the receipt-saving functionality works. If we need that, it should be in a test in a different part of the system.